independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "HubSpot\u2019s CRM is a system for managing customer relationships and storing data. The CRM custom connector provides access to CRM objects or groups of objects, which are represented as a map of property names to values.",

```

Comment:
@troystaylor Please remove all the unicodes from connector artifacts as unicodes may not be supported in all the Power Automate pages